third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -343,23 +353,16 @@ def arcs_kt_particles(
             )
             wasm_srcs.extend([src, wasm_annotations_file])
 
-        # Create a wasm library containing code for all the particles.
-        wasm_particle_lib = name + "-lib" + _WASM_SUFFIX
-        arcs_kt_native_library(
-            name = wasm_particle_lib,
-            srcs = wasm_srcs,
-            deps = wasm_deps,
-        )
-
         # Create a kt_native_binary that groups everything together.
         native_binary_name = name + _WASM_SUFFIX
         kt_native_binary(
```

Comment:
should you be using arcs_kt_native_library since you added it above?

Reply:
Yes, I should be using it, thank you.